migration/v12/migrate.go

```diff
@@ -0,0 +1,20 @@
+package v12
+
+import (
+	"github.com/jortel/go-utils/logr"
+	"github.com/konveyor/tackle2-hub/migration/v12/model"
+	"gorm.io/gorm"
+)
+
+var log = logr.WithName("migration|v11")
```

Comment:
the log name should be `migration|v12`

Reply:
Fixed.
We may want to correct this in the tooling.